perllib/FixMyStreet/Cobrand/UK.pm

```diff
@@ -392,5 +392,21 @@ sub link_to_council_cobrand {
     }
 }
 
+=head1 admin_problems
+
+Further restricts the results of self->problems to those which match the current
+user's from_body, unless the user is a superuser.
+
+=cut
+
+sub admin_problems {
```

Comment:
If you're on a council cobrand admin, you're already restricted to the `to_body` details. If you're on fixmystreet.com's admin, then I think that should probably be superuser only (which would be done by the previous change I imagine). If this is kept, I guess it should be in FixMyStreet.pm, but I'm not sure it's currently needed.